modules/openapi-generator/src/test/resources/3_0/enum_mapping_test.yaml

```diff
@@ -0,0 +1,77 @@
+openapi: 3.0.0
```

Comment:
this is an extract of the OpenAPI spec of https://developer.affinity.co/

Reply:
Apologies for the noise that introducing a new sample fixture produces, I couldn't find a sample OpenAPI spec that shows the issue I am trying to fix. If you know of one that has a discriminator which is an enum, please feel free to point me to it, then I might be able to use it instead and reduce the footprint of this PR a bit.